packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -212,11 +212,16 @@ export class DataVisualizerGraph
 		const result: Record<string, RootHandleNode> = {};
 		await Promise.all(
 			rootDataEntries.map(async ([key, value]) => {
-				const fluidObjectId = await this.registerVisualizerForHandle(value.handle);
-				result[key] =
-					fluidObjectId === undefined
-						? unknownObjectNode
-						: createHandleNode(fluidObjectId);
+				if (value.handle !== undefined) {
+					const fluidObjectId = await this.registerVisualizerForHandle(value.handle);
+					result[key] =
+						fluidObjectId === undefined
+							? unknownObjectNode
+							: createHandleNode(fluidObjectId);
+				} else {
+					console.error("containerData contains unknown data object!");
```

Comment:
Nit: I think we can be a bit more specific here. Maybe something like \`Container data includes a non-Fluid object under key ${key}. Cannot visualize.\`